tensorflow/compiler/xla/g3doc/operation_semantics.md

```diff
@@ -1359,9 +1359,9 @@ XlaBuilder supports these element-wise unary functions:
 
 <b>`Abs(operand)`</b> Element-wise abs `x -> |x|`.
 
-<b>`Atan2(operand)`</b> Element-wise arctangent of $X_i$, $Y_i$ - the angle
-measure in radians between the x-axis and a ray from the origin to a point (x,
-y) `x,y -> atan2(x,y)`.
+<b>`Atan2(operand)`</b> Element-wise arctangent of $Y_i$, $X_i$ - the angle
```

Comment:
From an internal review comment: Can you please also move this to the elementwise binary ops?

Reply:
Fixed it